### PR TITLE
Import ActionList during __init__

### PR DESCRIPTION
**Pull request by MrTeferi**
Could we add this to __init__ so you can call ps.ActionList() while doing photoshop actions?

### CODE REVIEW

**Comment by loonghao** [comment]
@MrTeferi 
You can use the following command line to format codes.

## setting up the dev environment
```
pip install poetry
```
```
poetry install
```
## format code
```
poetry run black photoshop
```
```
poetry run isort photoshop
```
## commit code
```shell
git add photoshop/api/__init__.py
```
https://github.com/commitizen-tools/commitizen
```
cz commit 
```

File: photoshop/api/__init__.py [hunk 1]
```diff
@@ -31,6 +32,7 @@
 __all__ = [  # noqa: F405
     "ActionDescriptor",
     "ActionReference",
+    "ActionList",
```
Comment (loonghao) [inline photoshop/api/__init__.py:35]: We also need to add `ActionList` in [`session.py`](https://github.com/loonghao/photoshop-python-api/blob/main/photoshop/session.py#L124)

**Review by loonghao — CHANGES_REQUESTED** [review]

**Review by MrTeferi — COMMENTED** [review]
Looks good to me!

**Comment by loonghao** [comment]
@MrTeferi We also need to update the current MR git commit message need to use `cz commit`
https://github.com/commitizen-tools/commitizen

because the CI will be using this commit message auto `bump-version` and auto-generate changelogs
image

**Pull request closed without merging**